charts/pulsar/templates/bookkeeper-statefulset.yaml

```diff
@@ -42,6 +42,48 @@ spec:
         {{- include "pulsar.template.labels" . | nindent 8 }}
         component: {{ .Values.bookkeeper.component }}
       annotations:
+        {{- if.Values.monitoring.datadog }}
+        ad.datadoghq.com/{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.component }}.check_names: |
+          ["openmetrics"]
+        ad.datadoghq.com/{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.component }}.init_configs: |
+          [{}]
+        ad.datadoghq.com/{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.component }}.instances: |
+          [
+            {
+              "prometheus_url": "http://%%host%%:%%port%%/metrics",
+              "namespace": "{{ .Values.datadog.namespace }}",
+              "metrics": [
+                "_*"
```

Comment:
metrics should be configurable, we can keep "_* " as default value. If users don't want to report all metrics, they can customize metrics to reports.